runtime/src/main/java/org/corfudb/runtime/view/Priority.java

```diff
@@ -0,0 +1,30 @@
+package org.corfudb.runtime.view;
+
+import lombok.AllArgsConstructor;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+/**
+ *
+ * Indicates the priority of LogUnit tasks.
+ *
+ * Created by Maithem on 6/24/19.
+ */
+@AllArgsConstructor
+public enum Priority {
+    LOW(0),
```

Comment:
Do we need 3 levels because each level can multiply the cases that we might need to handle. Can it just be 2 for now? NORMAL and HIGH?

Reply:
Done.

Reply:
> Here we are coupling the priority of the writes to the quota check directly.
> Can we add a policy as a handler of this priority? The default policy could be to check the quota.

I moved the priority level to the msg level, its much cleaner now. You can see #1974